### PR TITLE
minor improvement

### PR DESCRIPTION
**Pull request by volzkzg**
Use 1 lookup instead of 2.

### CODE REVIEW

File: src/core/votetime.rs [hunk 1]
```diff
@@ -56,19 +56,18 @@ impl WaitTimer {
         loop {
             select! {
                 settime = innersetter.recv() =>  {
-                let oksettime = settime.unwrap();
-                let notify = self.timer_notify.clone();
+                    let oksettime = settime.unwrap();
+                    let notify = self.timer_notify.clone();
```
Comment (TheWaWaR) [inline src/core/votetime.rs:60]: The extra indent is caused by cargo format?

File: src/core/voteset.rs [hunk 2]
```diff
@@ -198,8 +200,10 @@ impl VoteSet {
                             hash = h;
                         }
                         // inc the count of vote for hash
-                        if let Some(c) = votes_by_proposal.insert(hash, 1) {
-                            votes_by_proposal.insert(hash, c + 1);
+                        if let Some(c) = votes_by_proposal.get_mut(&hash) {
+                            *c += 1;
+                        } else {
+                            votes_by_proposal.insert(hash, 1);
```
Comment (TheWaWaR) [inline src/core/voteset.rs]: Better this way:
``` rust
*votes_by_proposal.entry(&hash).or_insert(0) += 1
```
* [Rust std doc](https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html)
* [Rust Playground](https://play.rust-lang.org/?gist=9f0d7928e5ee745174483244cd64f90c&version=stable&mode=debug)

Comment (TheWaWaR) [inline src/core/voteset.rs]: Better this way:
``` rust
*votes_by_proposal.entry(&hash).or_insert(0) += 1;
```
* [Rust std doc](https://doc.rust-lang.org/std/collections/hash_map/enum.Entry.html)
* [Rust Playground](https://play.rust-lang.org/?gist=9f0d7928e5ee745174483244cd64f90c&version=stable&mode=debug)

File: src/core/votetime.rs (hunk 1 above)
Comment (volzkzg) [reply src/core/votetime.rs:60]: No, these indents are added by hands. 
Currently cargo format can not handle things well inside macros.

File: src/core/voteset.rs (hunk 2 above)
Comment (volzkzg) [reply src/core/voteset.rs]: Nice method, man.

**Review by TheWaWaR — APPROVED** [review]

File: src/core/voteset.rs [hunk 3]
```diff
@@ -172,9 +172,7 @@ impl VoteSet {
             if let Some(h) = proposal {
                 hash = h;
             }
-            if let Some(c) = self.votes_by_proposal.insert(hash, 1) {
-                self.votes_by_proposal.insert(hash, c + 1);
-            }
+            *self.votes_by_proposal.entry(hash).or_insert(0) += 1;
```
Comment (u2) [inline src/core/voteset.rs]: or_insert(1)

**Review by u2 — APPROVED** [review]

**Pull request merged**